pages/community/ambassadors.tsx

```diff
@@ -12,6 +12,11 @@ import { ThirdwebNextPage } from "utils/types";
 const TRACKING_CATEGORY = "ambassadors_page";
```

Comment:
since this page moved you'll have to setup a redirect from `/ambassadors` -> `/community/ambassadors`

Reply:
Added redirect to redirects.js. Is there anywhere that needs it? 
```javascript
//Redirecting as ambassadors lives in community now
    {
      source: "/ambassadors",
      destination: "/community/ambassadors",
      permanent: false,
    },
```